### PR TITLE
Allow search-service to handle new hits type "favorited_search" 

### PR DESCRIPTION
**Pull request by nsharma123**
https://webarchive.jira.com/browse/WEBDEV-6341

### CODE REVIEW

File: src/models/hit-types/favorited_search-hit.ts
Comment (latonv) [inline src/models/hit-types/favorited_search-hit.ts:55]: 👍

File: package.json
Comment (latonv) [inline package.json:78]: Q: What was updated in the field parsers?

File: package.json
Comment (latonv) [inline package.json:3]: Reminder to undo alpha version before merging this.

**Review by latonv — CHANGES_REQUESTED** [review]
Thanks for the hit type addition!

Noted some testing changes that block merge. Otherwise LGTM aside from a couple of minor Qs/suggestions.

File: package.json
Comment (nsharma123) [reply package.json:78]: adding new mediatype type `search`.

File: yarn.lock
Comment (nsharma123) [reply yarn.lock:1]: yes, its mistake. reverting.

**Pull request merged**
